ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/panelcontainer/v1/panelcontainer/_cq_dialog/.content.xml

```diff
@@ -370,6 +370,25 @@
                                                     </table>
                                                 </items>
                                             </dorLayoutType>
+                                            <dorFieldStyling
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:resourceType="granite/ui/components/coral/foundation/form/select"
+                                                    emptyText="Select styling option"
+                                                    fieldLabel="Component Style"
+                                                    fieldDescription="These styles are derived from the Document of Record template (XDP) associated with this adaptive form."
+                                                    forceSelection="{Boolean}true"
+                                                    name="./dorFieldStyling">
+                                                <granite:rendercondition
+                                                        jcr:primaryType="nt:unstructured"
+                                                        sling:resourceType="granite/ui/components/renderconditions/featuretoggle"
+                                                        toggleName="FT_FORMS-16858"/>
```

Comment:
Ideally toggle should be present in cq-guide

Reply:
updated the toggle in cq-guides as well.